app/src/main/res/values/strings.xml

```diff
@@ -58,34 +58,34 @@
     <string name="notification_visibility_secret">Secretly visible notification</string>
     <string name="notification_visibility_placeholder">Visibility: %1$s</string>
     <string name="notification_color_placeholder">Notification color: %1$s</string>
-    <string name="notification_color_copied_to_clipboard">Notification color copied to clipboard!</string>
+    <string name="notification_color_copied_to_clipboard">Notification color copied to clipboard</string>
     <string name="date_placeholder">Date: %1$s</string>
-    <string name="power_history">Power history</string>
+    <string name="power_history">Power</string>
     <string name="power_disconnected">Power disconnected</string>
     <string name="power_connected">Power connected</string>
     <string name="ac_charging">AC charging</string>
     <string name="usb_charging">USB charging</string>
     <string name="wireless_charging">Wireless charging</string>
     <string name="error_occurred">An error occurred. Try again later.</string>
     <string name="channel_name">Vigilante usage notifications</string>
-    <string name="channel_description">Notifications are sent when privacy usage happens</string>
-    <string name="mic_being_used">Your microphone is being used!</string>
+    <string name="channel_description">Notifications are sent when usage occurs</string>
```

Comment:
```suggestion
    <string name="channel_description">Notifications are sent when privacy usage occurs</string>
```
I don't really understand what this word does here, so I am not sure it isn't needed.

Reply:
when one of the things vigilante notifies you about is triggered

Reply:
All other occurrences are just "usage", so going with the consistent option.